spinta/datasets/commands/load.py

```diff
@@ -105,28 +98,32 @@ def load(context: Context, resource: Resource, data: dict, manifest: Manifest):
     return resource
 
 
-def _check_unknown_keys(
-    model: Model,
-    keys: List[str],
-    data: Dict[str, Any],
-) -> None:
-    # XXX: Similar to spinta.types.helpers.check_no_extra_keys.
-    unknown = set(keys) - set(data)
-    if unknown:
-        raise MultipleErrors(
-            PropertyNotFound(model, property=name)
-            for name in sorted(unknown)
-        )
-
 
 @overload
 @commands.load.register(Context, Entity, dict, Manifest)
 def load(context: Context, entity: Entity, data: dict, manifest: Manifest):
     # XXX: https://gitlab.com/atviriduomenys/spinta/-/issues/44
     pkeys: List[str] = entity.pkeys or []
     if pkeys:
-        _check_unknown_keys(entity.model, pkeys, entity.model.properties)
-        entity.pkeys = [entity.model.properties[k] for k in pkeys]
+        unknown_properties=  []
+        entity.pkeys = []
+        for key in entity.pkeys:
```

Comment:
Wait, but how does this work? You are setting entity.pkeys to be an empty list, and then looping, when is it going in there?

Reply:
I guess it should be `for key in pkeys`?